noir-projects/aztec-nr/aztec/src/history/nullifier_inclusion.nr

```diff
@@ -26,10 +25,7 @@ fn _nullifier_inclusion(nullifier: Field, header: Header) {
     //     was included in the nullifier tree.
 }
 
-pub fn prove_nullifier_inclusion<TContext>(
-    nullifier: Field,
-    context: TContext
-) where TContext: ContextInterface {
+pub fn prove_nullifier_inclusion(nullifier: Field, context: PrivateContext) {
```

Comment:
Ah, we can simplify because we don't want to do this in public? Will we want to have this supported in public once the HEADERMEMBER opcode (or something similar) works?

Reply:
> Ah, we can simplify because we don't want to do this in public? 
Yes!
> Will we want to have this supported in public once the HEADERMEMBER opcode (or something similar) works?
From the discussion looks like headermember will only ever be able to get things from the latest header, so I don't think we'll ever want this. BTW the header was only used to get the block number.